packages/@aws-cdk/aws-apprunner/lib/observability-configuration.ts

```diff
@@ -0,0 +1,126 @@
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnObservabilityConfiguration } from './apprunner.generated';
+
+/**
+ * Properties of the AppRunner Observability Configuration
+ */
+export interface ObservabilityConfigurationProps {
+  /**
+   * Whether to enable X-Ray tracing on the observability configuration.
+   *
+   * @default true
+   */
+  readonly xrayTracing?: boolean;
+
+  /**
+    * The name for the Observability Configuration.
+    *
+    * @default - a name generated by CloudFormation
+    */
+  readonly observabilityConfigurationName?: string;
+}
+
+/**
+ * Attributes for the Observability Configuration
+ */
+export interface ObservabilityConfigurationAttributes {
```

Comment:
Why does this exist? This interface has the exact same properties as the interface defined immediately below it. This should likely be removed entirely. If this is just for the `fromAttributes()` method, it should be removed; in general though, if some method needs to take an interface, that interface should extend others where possible to avoid duplicating these same properties everywhere.